src/Command.hpp

```diff
@@ -0,0 +1,18 @@
+/**

```

Comment:
As specified by the [definition of done](https://github.com/R2D2-2018/R2D2-2018/wiki/Definition-of-Done), code should be documentated. Unfortunately, in the (abstract) classes you have included within your pull request, this is not the case. Please document the future function of your class methods as specified on the [wiki](https://github.com/R2D2-2018/R2D2-2018/wiki/Code-standard).